backend/LexBoxApi/Controllers/UserController.cs

```diff
@@ -80,20 +81,24 @@ public async Task<ActionResult<LexAuthUser>> RegisterAccount(RegisterAccountInpu
             PasswordStrength = UserService.ClampPasswordStrength(accountInput.PasswordStrength),
             IsAdmin = false,
             EmailVerified = emailVerified,
+            CreatedById = createdByAdmin ? jwtUser?.Id : null,
             Locked = false,
             CanCreateProjects = false
         };
         registerActivity?.AddTag("app.user.id", userEntity.Id);
         _lexBoxDbContext.Users.Add(userEntity);
-        if (jwtUser is not null && jwtUser.Projects.Length > 0)
+        if (jwtUser is not null && !createdByAdmin && jwtUser.Projects.Length > 0)
```

Comment:
I just changed this as I was already working on the PR. I'll let you resolve the comment.

Reply:
FYI, I added back the `jwtUser.Projects.Length > 0` check because in the future we might be issuing invitations that don't come attached to projects (e.g., inviting users to an organization).